### PR TITLE
Fix broken url in docs

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary

Easy navigation and easy reading for others

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @saintmalik thank you for your contribution.  the link you propose actually links back to a previous of the docs which would be confusing. Can you instead do a relative link to something in the current version of the docs.  So instead perhaps use https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/ or equivalent?

File: content/en/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Cluster administrator actions include:
 
 - [Draining a node](/docs/tasks/administer-cluster/safely-drain-node/) for repair or upgrade.
 - Draining a node from a cluster to scale the cluster down (learn about
-[Cluster Autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)
+[Cluster Autoscaling](https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)
```
[the remark this answers is not in the document]
Comment (saintmalik) [reply content/en/docs/concepts/workloads/pods/disruptions.md:52]: > the link you propose actually links back to a previous version of the docs which would be confusing. Can you instead do a relative link to something in the current version of the docs. So instead perhaps use /docs/tasks/run-application/horizontal-pod-autoscale/ or equivalent?

Okay, I will make the changes now.

[the remark this answers is not in the document]
Comment (saintmalik) [reply content/en/docs/concepts/workloads/pods/disruptions.md]: Done

File: content/en/docs/concepts/workloads/pods/disruptions.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Cluster administrator actions include:
 
 - [Draining a node](/docs/tasks/administer-cluster/safely-drain-node/) for repair or upgrade.
 - Draining a node from a cluster to scale the cluster down (learn about
-[Cluster Autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)
+[Cluster Autoscaling](/docs/tasks/run-application/horizontal-pod-autoscale/)
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/disruptions.md:52]: This is the wrong link. We need to link to a page that is about automatically managing the number of nodes in the cluster.

I would hyperlink to https://github.com/kubernetes/autoscaler/#readme at least for now.

Comment (saintmalik) [reply content/en/docs/concepts/workloads/pods/disruptions.md]: Made the changes

Comment (saintmalik) [reply content/en/docs/concepts/workloads/pods/disruptions.md]: I have made the changes

**Comment by sftim** [comment]
/sig autoscaling
I suppose

/lgtm

File: content/en/docs/concepts/workloads/pods/disruptions.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Cluster administrator actions include:
 
 - [Draining a node](/docs/tasks/administer-cluster/safely-drain-node/) for repair or upgrade.
 - Draining a node from a cluster to scale the cluster down (learn about
-[Cluster Autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)
+[Cluster Autoscaling](https://github.com/kubernetes/autoscaler/#readme)
```
Comment (bradtopol) [inline content/en/docs/concepts/workloads/pods/disruptions.md:52]: Link works. Great choice!

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**